src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/RazorSemanticTokensEndpoint.cs

```diff
@@ -10,6 +9,7 @@
 using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
 using OmniSharp.Extensions.LanguageServer.Protocol.Document;
 using System.Diagnostics;
```

Comment:
Weird that we have `using System.*` on both sides of the MS using's, I assume you ran "Sort using's" against this?

Reply:
I used the code action to auto-add usings and it automatically placed them there, I'll run Sort Usings on the file though to resolve 😄